src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1185 @@
+    if (vdExpr.getVars().size() != 1) {
+      // Number of variables declared in this expression is not one.
+      return null;
+    }
+    // Get the name of the variable being declared.
+    String varName = vdExpr.getVars().get(0).getId().getName();
+
+    // Check if the map contains a value found from a passing assertion.
+    if (primitiveValues.containsKey(varName)) {
+      String value = primitiveValues.get(varName);
+      return rhsAssignWithValue(vdExpr, value);
+    } else {
+      // The map does not contain a value for this variable.
+      return null;
+    }
+  }
+
+  /**
+   * Return a variable declaration statement that sets the right hand side of a variable declaration
+   * to the value that is passed in.
+   *
+   * @param vdExpr variable declaration expression representing the current statement to simplify
+   * @param value value that will be assigned to the variable being declared
+   * @return a {@code Statement} object representing the simplified variable declaration expression
+   *     if the type of the variable is a primitive and a value has been previously calculated.
+   *     Returns {@code null} if more than one variable is declared in the {@code
+   *     VariableDeclarationExpr}.
+   */
+  private static Statement rhsAssignWithValue(VariableDeclarationExpr vdExpr, String value) {
+    if (vdExpr.getVars().size() != 1) {
+      // Number of variables declared in this expression is not one.
+      return null;
+    }
+
+    // Copy the variable declaration expression.
+    VariableDeclarationExpr exprCopy = (VariableDeclarationExpr) vdExpr.clone();
+
+    // Obtain a reference to the variable declaration.
+    List<VariableDeclarator> vars = exprCopy.getVars();
+    VariableDeclarator vd = vars.get(0);
+
+    // Based on the declared variable type, set the right hand to the value
+    // that was passed in.
+    Type type = exprCopy.getType();
+    if (type instanceof PrimitiveType) {
+      switch (((PrimitiveType) type).getType()) {
+        case Boolean:
+          vd.setInit(new BooleanLiteralExpr(Boolean.parseBoolean(value)));
+          break;
+        case Byte:
+          vd.setInit(new IntegerLiteralExpr(value));
+          break;
+        case Char:
+          vd.setInit(new CharLiteralExpr(value));
+          break;
+        case Short:
+          vd.setInit(new IntegerLiteralExpr(value));
+          break;
+        case Int:
+          vd.setInit(new IntegerLiteralExpr(value));
+          break;
+        case Long:
+          vd.setInit(new LongLiteralExpr(value));
+          break;
+        case Float:
+          vd.setInit(new DoubleLiteralExpr(value));
+          break;
+        case Double:
+          vd.setInit(new DoubleLiteralExpr(value));
+          break;
+      }
+    } else {
+      // Set right hand side to null.
+      vd.setInit(new NullLiteralExpr());
+    }
+    // Create a new statement with the simplified expression.
```

Comment:
Here is another comment that doesn't add much to the code immediately below it.  Variable `exprStmt` will be returned, so giving it a name containing `result` would be more informative.